packages/strapi-plugin-content-manager/admin/src/components/Edit/index.js

```diff
@@ -18,8 +18,9 @@ import {
 // ./node_modules/strapi-helper-plugin/lib/src
 // or strapi/packages/strapi-helper-plugin/lib/src
 import Input from 'components/InputsIndex';
-import InputJSONWithErrors from 'components/InputJSONWithErrors';
-import WysiwygWithErrors from 'components/WysiwygWithErrors';
+
+import InputJSONWithErrors from '../InputJSONWithErrors';
+import WysiwygWithErrors from '../WysiwygWithErrors';
```

Comment:
We should rename this component... WYSIWYG is an acronym.

Reply:
Well I'm not sure it's the purpose of this PR

Reply:
Anyway, we won't do it but it's not readable enough, IMO.